pkg/util/alibaba/alibaba.go

```diff
@@ -33,6 +33,9 @@ func IsRunningOn() bool {
 
 // GetHostAlias returns the VM ID from the Alibaba Metadata api
 func GetHostAlias() (string, error) {
+	if isEnabled := config.IsCloudProviderEnabled(CloudProviderName); !isEnabled {
```

Comment:
You don't need to create a variable here, just `if !config.IsCloudProviderEnabled(CloudProviderName) {`. Same for all the other condition